src/client/src/commons/form/profile/components/profileLayers/profileLayers.js

```diff
@@ -147,6 +149,14 @@ const ProfileLayers = props => {
     minHeight: "10em",
   };
 
+  if (!layers?.data || !lithoDescQuery.isSuccess) {
```

Comment:
PP:
```suggestion
  if (lithoDescQuery.isLoading || lithoDescQuery.isError) {
```

Reply:
Warum findest du das besser?

Reply:
Ich hatte nicht bemerkt, dass die `layers` und `lithoDescQuery` api überprüft wird 🤦 . Ich bevorzuge conditions, die kein `!` not brauchen, wenn möglich, da einfacher zu lesen. Aber ich finde in diesem Fall hat sichs erledigt.